internal/fs/writebuffer/buffer.go

```diff
@@ -0,0 +1,18 @@
+package writebuffer
```

Comment:
the filename and interface name has to match.

Make the package name as buffer.
interface as WriteBuffer and file name as writebuffer.go.

memory buffer as Inmemorywritebuffer.go.

Reply:
should filename and struct name also match? shall I rename MemoryBuffer to InMemoryWriteBuffer?

Reply:
yes. atleast to begin with they should match. If we add more structs in the same file in future, its okay

Reply:
Also updated the comment, please use underscores in filenames. thats the pattern we are following in gcsfuse.